server/index.js

```diff
@@ -112,6 +118,32 @@ app.use('/api/add/byfeedurl', async (req, res) => {
     res.send(response)
 })
 
+// ------------------------------------------------
+// ------------ API to get comments for episode ---
+// ------------------------------------------------
+app.use('/api/comments/byepisodeid', async (req, res) => {
+    let episodeId = req.query.id;
+    const response = await api.episodesById(episodeId, false);
+
+    const socialInteract = response.episode.socialInteract && response.episode.socialInteract.filter((si) => si.protocol === 'activitypub');
+
+    if(!socialInteract && socialInteract.lenght >= 0) {
```

Comment:
@dellagustin FYI. It looks like the length property got typoed here ( `.lenght` )

Reply:
@ericpp good catch! I thought I tested that.
I'll fix this.

Reply:
Having some trouble running my local setup now: https://github.com/Podcastindex-org/web-ui/issues/231

Reply:
So, it turns out I made two mistakes here that cancelled each other, so it still works out.
It still works when `socialInteract `is there and with one or more items with `activitypub`.
So, it is not urgent.
I'll transfer to an issue and take care later.

Reply:
There you go, https://github.com/Podcastindex-org/web-ui/issues/232.
I'll follow up from there and resolve this comment.